packages/app/src/app/overmind/namespaces/deployment/actions.ts

```diff
@@ -219,3 +222,53 @@ export const aliasDeployment = async (
     });
   }
 };
+
+export const deployWithGitHubPages = async ({
+  effects,
+  actions,
+  state,
+}: Context) => {
+  const sandbox = state.editor.currentSandbox;
+
+  if (!sandbox) {
+    return;
+  }
+
+  state.deployment.deploying = true;
+
+  try {
+    await effects.githubPages.deploy(
+      sandbox,
+      state.deployment.githubSite.ghLogin
+    );
+    state.deployment.deploying = false;
+
+    state.deployment.building = true;
+    await effects.githubPages.getLogs(sandbox.id);
+    state.deployment.githubSite.ghPages = true;
+    effects.notificationToast.success('Sandbox Deploying');
```

Comment:
So If I understand correctly it's deploying at this point?

Reply:
Yes! We started the process